src/Microsoft.OData.Edm/Csdl/CsdlJsonWriter.cs

```diff
@@ -72,6 +90,27 @@ private void WriteCsdlStart()
             }
         }
 
+        /// <summary>
+        /// CSDL JSON Document Object asynchronously
+        /// </summary>
+        /// <returns></returns>

```

Comment:
Same here. Complete the doc string.

Reply:
@ElizabethOkerio Thanks for pointing this out. I have fix this.